src/Network/Response.php

```diff
@@ -417,6 +430,20 @@ class Response
      */
     public function __construct(array $options = [])
     {
+        if (isset($options['streamTarget'])) {
+            $this->_streamTarget = $options['streamTarget'];
+        }
+        if (isset($options['streamMode'])) {
+            $this->_streamMode = $options['streamMode'];
```

Comment:
What is the purpose of these options? Do you have any scenarios where you would want to create a `r` stream in a response?